packages/documentation/src/stories/foundation/accessibility/accessibility.docs.mdx

```diff
@@ -12,6 +12,22 @@ import liveRegionBadAngularSample from './liveregion-bad-angular.sample.html?raw
   TODO: write accessibility page
 </div>
 
+The following article of the Swiss postal law is an official ruling concerning accessibility issues.
```

Comment:
I don't think this is expected for the reader here. @alizedebray @gfellerph @oliverschuerch What do you think? It could be put at the bottom of the page perhaps?

Reply:
I'd like to have that quote somewhere and to me, it's not horribly misplaced at the beginning as an intro to keep in mind while reading the guide.

I'd just delete the introduction sentence and shorten the quotation a bit.

```suggestion
```